kt/godot-library/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ dependencies {
     // added here as a transitive dependency so the user can use reflection
     // we need to add it here so reflection is available where the code is loaded (Bootstrap.kt) otherwise it will not work
     api(kotlin("reflect", version = libs.versions.kotlin.get()))
-    implementation("com.utopia-rise:common:$fullGodotKotlinJvmVersion")
+    api("com.utopia-rise:common:$fullGodotKotlinJvmVersion")
```

Comment:
Why should this be a transitive dependency?

Reply:
Some of those needs to be usable in the user code, so `implementation` is not enough, otherwise I can't even compile the tests. It technically exposes more things than before to the user, but they are all utilities without any global state, so there is no risk. Also, I can still move a few of them to the new Internal module in my next PR if we really want to hide some. But otherwise, everything in common is supposed to be 100% public.